django/db/models/fields/related_objects.py

```diff
@@ -0,0 +1,315 @@
+"""
+"Rel objects" for related fields.
+
+"Rel objects" (for lack of a better name) carry information about the relation
+modeled by a related field and provide some utility functions. They're stored
+in the ``rel`` attribute of the field.
```

Comment:
Yes, that was the first step in making the rels like fields. There have been other small changes to that direction, too.

I think your suggestion is a good one, we already claim the rel objects are actually fields, so why not do it here, too.